md/admin-application-resources-list.md

```diff
@@ -0,0 +1,106 @@
+# Resources list in the Bonita Administrator Application
+
+This page defines the term "Resources" and explains what a user with the _Administrator_ profile in the Bonita Living Application can see and do about resources.  
+
+_Administrators_ can view a list of the resources filtered by type, search a resource by its name, install new resources, update existing resources, or delete resources.  
+
+Here is a view of the page:
+![Administrator Resources](images/UI2021.1/admin-application-resources-list.png)<!--{.img-responsive}-->
+
+
+## Resource definition
+
+A resource is deployed in the Bonita Administrator Application to create the User Interface of a Living Application: it is a page, a layout, a theme, a REST API extension, but also a form that will be used by several processes so it is better to store it at platform level than within the process .bar file.  
+
+A resources is packaged as a .zip archive that contains a `page.properties` file, a resources directory and an index file.  
+* The `page.properties` file contains the metadata for the page.
+
+For example: 
+```
+#The name must start with 'custompage_'
+name=custompage_layout
+displayName=Application layout page
+description=This is a layout page dedicated to new born living applications. It is created and editable using the UI designer. It allows to display an horizontal menu, and an iframe. The menu allows to target some pages and the iframe define the area to display those targeted pages.
+resources=[GET|living/application,GET|living/application-page,GET|living/application-menu]
+contentType=layout
+```
+
+* The `resources` directory contains all the public files of your resource (for example `index.md`, images, Javascript files, CSS files).  
+
+* The zip archive must contain at least one of the following index files:
+   * An `index.html` file in the `resources` directory
+   * An `Index.groovy` class at the root of the archive with, optionally, libraries
+
+If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.  
+
+## Permissions for resources
+
+If you are using [REST API authorization](rest-api-authorization.md) and your resource needs the Web REST API, 
+you need to specify the permissions that a user needs to have in order to access those API. 
+These permissions are defined in the `page.properties` file. If your resource is a page written in Groovy and uses the Bonita Engine Java APIs, you do not need to specify permissions.
+For each REST resource accessed in other resources, specify the authorization needed for each method used. 
+You can find examples of the default resources in [`resources-permissions-mapping.properties`](BonitaBPM_platform_setup.md).
+
```

Comment:
```suggestion

When a user is given access to a page/layout because they are in a profile that contains it or that is mapped to an application that contains the page/layout, then this user is granted, upon login, the permissions associated to these REST resources (see [REST API authorizations](rest-api-authorization.md) for more details).  

```